soda/core/tests/data_source/test_profile_columns.py

```diff
@@ -86,10 +91,18 @@ def test_profile_columns_text(data_source_fixture: DataSourceFixture):
     profiling.pop("dataSource")
     profiling.pop("table")
 
+    test_data_source = os.environ.get("test_data_source")
+
+    uppercase_column_name_databases = "snowflake"
```

Comment:
Are you this is only snowflake? I suspect on db2 and oracle  (and potentially sqlserver) we also have similar issues

Reply:
Hey @vijaykiran it does indeed seem that `db2` and `oracle` use uppercase column names as well, but we don't have unit tests for those yet right? I have added them to the list so that once we do those database types will not fail CI. 

For `sqlserver` we do have unit tests (and they passed), so it seems to case sensitive.

Thanks!

Reply:
we don’t have CI for oracle or db2, only local ones. Oracle CI will be added this sprint. For db2 the container creation is too slow, and we don’t have enough production usage yet. 

For now, I’ll merge it. If oracle CI fails, I’ll let you know.